Headers/DebugServer2/GDBRemote/DummySessionDelegateImpl.h

```diff
@@ -12,13 +12,15 @@
 #define __DebugServer2_GDBRemote_DummySessionDelegateImpl_h
 
 #include "DebugServer2/GDBRemote/SessionDelegate.h"
+#include "DebugServer2/Host/FileIO.h"
 
 namespace ds2 {
 namespace GDBRemote {
 
 class DummySessionDelegateImpl : public SessionDelegate {
 protected:
   bool _secure;
+  Host::FileIO _fileIO;
```

Comment:
This won't break with multiple open and close calls. A FileIO object isn't just one file (which is why I didn't call the class File).